app/src/main/kotlin/cz/covid19cz/erouska/ui/help/HelpFragment.kt

```diff
@@ -26,24 +30,41 @@ class HelpFragment : BaseFragment<FragmentHelpBinding, HelpVM>(R.layout.fragment
                 HelpCommandEvent.Command.GO_BACK -> goBack()
             }
         }
+
+        isFullscreen = arguments?.getBoolean("fullscreen") == true
+        type = when (arguments?.getString("type")) {
```

Comment:
With safeArgs, you can extract fragment arguments in a type-safe way. It even provides delegate `by navArgs()`

Reply:
Updated